velox/exec/Window.cpp

```diff
@@ -102,6 +102,20 @@ Window::Window(
 void Window::createWindowFunctions(
     const std::shared_ptr<const core::WindowNode>& windowNode,
     const RowTypePtr& inputType) {
+  auto constantArg = [&](const core::TypedExprPtr arg) -> const VectorPtr {
```

Comment:
Maybe we could refactor this into a helper function to avoid copy-paste (in a follow-up PR).